mapdata/WarcraftLegacies/AbilityData/Skin/1093873729.json

```
@@ -72,13 +72,13 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Harvests lumber from trees."
+      "Value": "Harvest Gold from gold mines"
     },
     {
       "Level": 1,
       "Id": 829781345,
       "Type": 3,
-      "Value": "Returns carried lumber."
+      "Value": "Returns carried ressources."
```

Resources is a bit confusing here because gold is the only resource!
```suggestion
      "Value": "Returns carried gold."
```